build.gradle

```diff
@@ -1,6 +1,11 @@
 buildscript {
   ext.versions = [
-    'kotlin': '1.3.60'
+    'kotlin': '1.3.70',
+    'serializationRuntime': '0.20.0',
+    'retrofit': '2.7.2',
```

Comment:
This needs to stay at 1.6.4 to remain compatible with people using Java 6 or Android API <21

Reply:
Updated.